aries-site/src/pages/components/table.mdx

```diff
@@ -175,6 +175,16 @@ accessible in a tool specifically designed for analysis.
   - Align it horizontally with the rest of its layout context. Typically, this is aligned to the start of the content as opposed to stretched.
   - Align any header above the table, including heading, search, filter, actions, in the same way. This keeps an actions menu or button in the context of the table and prevents them from being orphaned. However, don't force the header above the table to align its width to the table, as that might compress the contents of the header too much.
 
+  ### Setting the height of a table
+  
+- Tables should be placed within the main content of the page, it is reccommended to use
```

Comment:
Wording suggestion:


- Let the data set the height of the Table. The Table will grow in height as the length of data increases.
- If it is expected a user will want to navigate to a specific portion of the data, pagination should be used. Generally, it is recommended that a page has a `step` of 20 records. For more information on pagination, read [Pagination guidance](https://design-system.hpe.design/components/pagination).
- Do not restrict the height of a table if the page has available space. This may introduce unnecessary scrolling.
- If the height of a Table needs to be restricted, set `overflow=“auto”` on its parent Box to allow the user to scroll through the data.